apps/wallet-mobile/src/components/CopyButton.tsx

```diff
@@ -29,17 +33,32 @@ export const CopyButton = ({value, onCopy, children, style}: CopyButtonProps) =>
   )
 }
 
-const AnimatedCopyButton = ({
+export const AnimatedCopyButton = ({
   onCopy,
   children,
   style,
   isCopying,
-}: Omit<CopyButtonProps, 'value'> & {isCopying: boolean}) => {
-  const {colors} = useStyles()
+  message,
+}: Omit<CopyButtonProps, 'value'> & {isCopying: boolean; message?: string | null | undefined}) => {
+  const {styles, colors} = useStyles()
 
   return (
     <TouchableOpacity onPress={onCopy} disabled={isCopying} testID="copyButton" style={style}>
-      {isCopying ? <Icon.CopySuccess size={26} color={colors.gray} /> : <Icon.Copy size={26} color={colors.gray} />}
+      {isCopying ? (
```

Comment:
@michaeljscript Thanks for the review.
The component is used in few places where it is not passing any string message(as it was earlier), so if we combine the two conditions, this line <Icon.CopySuccess size={26} color={colors.gray} /> will not work for calls thats have an empty message. 
I am not sure if this is the best way. Please advise if it could be made simpler.